src/python/WMCore/WMSpec/WMWorkload.py

```diff
@@ -749,7 +749,7 @@ def getAcquisitionEra(self, taskName=None):
         """
         if taskName and isinstance(self.acquisitionEra, dict):
             return self.acquisitionEra.get(taskName, None)
-        return self.acquisitionEra
+        return
```

Comment:
Not sure at which stage we would need this info, but for ReqMgr1 requests it would always be None then